components/molecules/Searchbar/index.js

```diff
@@ -19,6 +19,10 @@ const SearchBar = ({ doFetchEvent }) => {
   const handleClick = () => {
     const sendCountry = location.country.toLowerCase().replace(' ', '-');
     const sendCity = location.city === 'Abuja Federal Capital Territory' ? 'abuja' : location.city.toLowerCase().replace(' ', '-');
+    localStorage.setItem('userSearchCountry', sendCountry);
+    localStorage.setItem('userSearchCity', sendCity);
+    localStorage.setItem('userEventType', interest);
```

Comment:
I guess these localStorage variables are going to be reused?

Reply:
Yes. Check the Calendar organism component